src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,116 @@
+package duke.storage;
+
+import duke.task.*;
```

Comment:
Try not to use wildcards and instead state all explicit imports.

Reply:
yes